client/layout/guided-tours/style.scss

```diff
@@ -337,3 +337,10 @@ $guided-tour-step_border: 2px solid white;
 .guided-tours__step-dark.guided-tours__step-pointing-left-top::before {
 	left: -8.5px;
 }
+
+.card.guided-tours__step {
+	@include breakpoint-deprecated( '<480px' ) {
+		top: 90px !important;
```

Comment:
This feels it needs to be improved for several reasons:

* How did we determine that `90px` is the right amount? Why not 95? Why not 85?
* Using `!important` is strongly discouraged. 
* The position of guided tour steps is automatically calculated based on the given settings and constraints. If we need to override those calculations, then the mechanism for calculation is broken. I'd suggest looking at the root of the problem and fixing it there.

Reply:
>     * The position of guided tour steps is automatically calculated based on the given settings and constraints. If we need to override those calculations, then the mechanism for calculation is broken. I'd suggest looking at the root of the problem and fixing it there.

What you have said makes sense, I think we have to use what it available. I will try to rework the same with the options available https://github.com/Automattic/wp-calypso/blob/trunk/client/layout/guided-tours/docs/API.md#props-1

Reply:
I am now using the `beside` option instead of the `above`, Here is the test for responsiveness:
![](https://d.pr/i/NkDJen+)
[Direct link](https://d.pr/i/NkDJen)

@tyxla